src/sagemaker/model.py

```diff
@@ -285,10 +285,6 @@ def transformer(self, instance_count, instance_type, strategy=None, assemble_wit
             max_payload (int): Maximum size of the payload in a single HTTP request to the container in MB.
             tags (list[dict]): List of tags for labeling a transform job. If none specified, then the tags used for
                 the training job are used for the transform job.
-            role (str): The ``ExecutionRoleArn`` IAM Role ARN for the ``Model``, which is also used during
```

Comment:
I am not being able to understand the connection of removal of these lines to this PR. Are these changes intentional?

Reply:
Yes. There's no `role` or `model_server_workers` parameters in this method. (The docstring was probably just copied from the estimator `transformer` method, which does have these parameters)